snapcraft/cli/_options.py

```diff
@@ -179,13 +179,6 @@ def __repr__(self):
         envvar="SNAPCRAFT_ENABLE_EXPERIMENTAL_EXTENSIONS",
         supported_providers=["host", "lxd", "managed-host", "multipass"],
     ),
-    dict(
-        param_decls="--enable-experimental-package-repositories",
```

Comment:
I am hesitant about this potentially breaking people, but it is an experimental feature after all